code/modules/almayer/machinery.dm

```diff
@@ -295,3 +295,85 @@
 	new /obj/item/clothing/under/shorts/red(src)
 	new /obj/item/clothing/under/shorts/blue(src)
 	new /obj/item/clothing/under/shorts/green(src)
+
+// --- CIC Map Table --- //
+
+/obj/structure/machinery/prop/almayer/CICmap/table
```

Comment:
this is a prop, why not make it a subtype of existing tables so we don't copypaste code?

Reply:
Is it any worse than the other Map Tables that are also props?

Reply:
Is it any worse than the other map tables that are also props?